plugins/native/module/vtkF3DSplatReader.cxx

```diff
@@ -4,15 +4,20 @@
 #include <vtkCellData.h>
 #include <vtkCommand.h>
 #include <vtkDemandDrivenPipeline.h>
+#include <vtkFileResourceStream.h>
 #include <vtkFloatArray.h>
 #include <vtkIdTypeArray.h>
 #include <vtkInformation.h>
 #include <vtkInformationVector.h>
 #include <vtkNew.h>
+#include <vtkObjectFactory.h>
 #include <vtkPointData.h>
 #include <vtkPoints.h>
 #include <vtkPolyData.h>
+#include <vtkResourceStream.h>
```

Comment:
you need to put this include behind a 

```
#if VTK_VERSION_NUMBER >= VTK_VERSION_CHECK(9, 2, 20221216)
```

Reply:
> now it fails the test for VTK 9.2, since it does not include vtkResourceStream.h, but I still reference it in the vtkF3DSplatReader.
> In vtkF3DSPZReader, it does not include vtkResourceStream.h, but still reference it! Why would this file pass the CI check?

Ha right, sorry I was wrong. You can look at how it is done for SPX here: 
 - https://github.com/f3d-app/f3d/blob/master/plugins/native/CMakeLists.txt#L199
 - https://github.com/f3d-app/f3d/blob/master/plugins/native/module/CMakeLists.txt#L7
 
 Lets do the same for splat. I mean that Splat reading will not be available for VTK 9.2.6 users but I think this is fine.